app/src/main/java/com/greenfox/kalendaryo/ChooseAccountActivity.java

```diff
@@ -104,5 +107,9 @@ public void emailChanged(String email) {
         accountNamesView.setAdapter(accountAdapter);
     }
 
-
```

Comment:
was this necessary? this way we cannot go back to choosing calendars, if we missed one, because it goes back to the MainActivity